### PR TITLE
Add a Pause state to the ReplayableVideoAsset

### PR DESCRIPTION
**Pull request by dobromir-hristov**
Bug/issue #, if applicable: 107269394

## Summary

Adds a Pause text and stops hiding the control button, when the native action buttons are hidden.

## TODO

Add a Pause icon, if necessary.

## Dependencies

NA

## Testing

Test an autoplay video and a normal video, both desktop and mobile to see with and without native controls.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

**Comment by dobromir-hristov** [comment]
@swift-ci test

**Review by marinaaisa — APPROVED** [review]
LGTM! Thanks Dobri :)

**Review by marinaaisa — APPROVED** [review]
I tested the Chrome issue and it's now fixed. Great!

**Comment by dobromir-hristov** [comment]
@swift-ci test

File: tests/unit/components/ReplayableVideoAsset.spec.js [hunk 1]
```diff
@@ -6,13 +6,14 @@
  *
  * See https://swift.org/LICENSE.txt for license information
  * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
-*/
+ */
```
Comment (dobromir-hristov) [inline tests/unit/components/ReplayableVideoAsset.spec.js:9]:
```suggestion
*/
```

**Comment by dobromir-hristov** [comment]
@swift-ci test

**Pull request merged**